src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainTestConfig.scala

```diff
@@ -182,7 +183,11 @@ class HomesteadToDaoAt5 extends BlockchainTestConfig {
 }
 
 class Eip158Config extends BlockchainTestConfig {
+  override val frontierBlockNumber = -1
```

Comment:
why -1 and not just 0?

Reply:
I guess they could be `0`, but then we'd either have to bump the relevant fork numbers to `1` or modify `EvmConfig.forBlock` method (which doesn't handle ties very well). But I'd also argue that `-1` is more explicitly stating that a given fork is not occurring in the tested chain.